src/components/EventsTimeline/stories/basic.story.mdx

```diff
@@ -107,11 +107,11 @@ const toTimelines = ({ id }: CogniteEventForTimeline) => {
 ### Ruler
 
 ```typescript
-const onChange = (
-  _: React.SyntheticEvent,
-  date: number,
-  tlEvents?: CogniteEventForTimeline[]
-) => console.log('Date - ', new Date(date), 'Events – ', tlEvents);
+const onChange = ({
+  event: React.SyntheticEvent,
+  timestamp: number,
+  timelineEvents?: CogniteEventForTimeline[]
+}) => console.log('Date - ', new Date(timestamp), 'Events – ', timelineEvents);
```

Comment:
```
export const onChange = ({
  timestamp,
  timelineEvents
}: TimelineRulerChangeProps) 
=> console.log('Date - ', new Date(timestamp), 'Events – ', timelineEvents);
```
right?

Reply:
what I meant here is that onChange takes an object as a parameter now, and this syntax
`prop: proptype,` is not valid anymore. please, have a look more attentively here again

Reply:
yeah, fixed now